### PR TITLE
update Kubeflow roadmap to show delivery of v1.6

### PR DESCRIPTION
**Pull request by jbottum**

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
-## Kubeflow 1.6 Release, Due: July 2022
+## Kubeflow 1.7 Release - planning now in process 
+* Please join the Kubeflow Community Meetings on Tuesdays for updates and for opportunities to contribute.  
+
+## Kubeflow 1.6 Release, Delivered: September 2022
 * Kubeflow 1.6 [milestones and timelines](https://github.com/kubeflow/community/pull/558)
 
 #### Themes
 * Kubernetes 1.22 support
+* Kubeflowo Pipelines v2 (alpha)
```
Comment (xin-hao-awx) [inline ROADMAP.md:11]: typo `Kubeflow`

File: ROADMAP.md [hunk 2]
```diff
@@ -1,18 +1,21 @@
 # Kubeflow Roadmap
 
-## Kubeflow 1.6 Release, Due: July 2022
+## Kubeflow 1.7 Release - planning now in process 
+* Please join the Kubeflow Community Meetings on Tuesdays for updates and for opportunities to contribute.  
+
+## Kubeflow 1.6 Release, Delivered: September 2022
 * Kubeflow 1.6 [milestones and timelines](https://github.com/kubeflow/community/pull/558)
```
Comment (annajung) [inline ROADMAP.md:7]:
```suggestion
## Kubeflow 1.6 Release, Delivered: September 2022
* Kubeflow 1.6 [milestones and timelines](https://github.com/kubeflow/community/tree/master/releases/release-1.6)
```

File: ROADMAP.md (hunk 1 above)
Comment (annajung) [inline ROADMAP.md:11]:
```suggestion
* Kubeflow Pipelines v2 (alpha)
```

**Comment by annajung** [comment]
I had this PR https://github.com/kubeflow/kubeflow/pull/6508 open for awhile, but I'm in favor of keeping yours and closing mine. Just added the additional feedback to change one of the old links.

File: ROADMAP.md (hunk 1 above)
Comment (jbottum) [reply ROADMAP.md:11]: thanks!

**Comment by jbottum** [comment]
@DomFleischmann would you like to review and provide a /lgtm

**Review by DomFleischmann — APPROVED** [review]
/lgtm

**Pull request merged**